mungegithub/mungegithub.go

```diff
@@ -107,6 +107,8 @@ func main() {
 			if err := mungers.InitializeMungers(&config.Config, &config.Features); err != nil {
 				glog.Fatalf("unable to initialize mungers: %v", err)
 			}
+			config.HookHandler.Initialize()
+			go config.HookHandler.Listen()
```

Comment:
No, we need the webhooks to listen to status (unfortunately). I'll move the listen.